src/WinSparkleDotNet/WinSparkle.cs

```diff
@@ -0,0 +1,483 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace WinSparkleDotNet
+{
+    sealed public class WinSparkle
+    {
+        ////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
They aren't standard, that is just the template from my last internship/enterprise code convention.